index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Html CSS Quiz</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+	<div class="main-content clear">
+		<article class="card card-1">
+			<header class="card-header">
+				<h1>White T-Shirt</h1>
+			</header>
+				<section>
+					<img src="img/white.png">
```

Comment:
Your image is displayed as block, but it turns out Requirement 5-i asks for the image to be contained by a block element. :/